tunder/lib/src/http/route_entry.dart

```diff
@@ -356,36 +333,30 @@ class RouteEntry implements RouteDefinitions {
 
     return method.parameters.map((param) {
       var type = param.type.reflectedType;
-      if (type == Request) return request;
-      if (_isPrimitive(type)) {
-        return paramValue(
-          param.simpleName,
-          request: request,
-          castTo: type,
-        );
-      }
 
-      return container.get(param.type.reflectedType);
+      if (type == Request) {
+        return request;
+      } else if (_isPrimitive(type)) {
+        return paramValue(param.simpleName, request: request, castTo: type);
+      } else {
+        return container.get(param.type.reflectedType);
+      }
```

Comment:
Oh, I didn't see this before, but I always avoid `else if` and `else` statements. I don't use `else` in my code and I'm an advocate against else statements. Could you rever this change please?